actionview/test/ujs/public/vendor/jquery-2.2.0.js

```diff
@@ -9639,7 +9639,7 @@ jQuery.fn.extend( {
 	// 2) For the hidden or detached element
 	// 3) For body or html element, i.e. in case of the html node - it will return itself
 	//
-	// but those exceptions were never presented as a real life use-cases
+	// but those exceptions were never presented as a real life use cases
```

Comment:
This is the jquery source file. We should not be editing it IMO.

Reply:
Oh, that was an overlook, didn’t realize it.

Don’t have a computer this weekend, would you mind opening a PR reverting this edit? Please cc me if you do.